pkgs/development/lua-modules/overrides.nix

```diff
@@ -160,6 +156,32 @@ in
       '';
   });
 
+  fzf-lua = prev.fzf-lua.overrideAttrs (oa: {
+    doCheck = true;
+    checkInputs = [
+      fd
+      fzf
+      ripgrep
+    ];
+    nativeCheckInputs = [
+      neovim-unwrapped
+    ];
+    checkPhase = ''
+      runHook preCheck
+      export HOME=$(mktemp -d)
+
+      # Linking the dependencies since makefile wants to clone them each time
+      # for `make deps`
+      mkdir -p deps
+      ln -s ${vimPlugins.mini-nvim} deps/mini.nvim
```

Comment:
not sure how mini works but looks like it only uses mini-test so vimPlugins.mini-test could alleviate the closure.

Reply:
I did that originally, but then changed to copy what upstream does in their `make deps` https://github.com/ibhagwan/fzf-lua/blob/0a3b70feb05879a8001c51f7a2a42fa52a9e552c/scripts/minimal_init.lua to match their behavior